Formula/conan.rb

```diff
@@ -33,16 +42,41 @@ class Conan < Formula
     sha256 "e043c8d32527607223652021ff648fbb394d5e19cba9f1a698670b338c9d782b"
   end
 
+  resource "cryptography" do
+    url "https://pypi.python.org/packages/a9/5b/a383b3a778609fe8177bd51307b5ebeee369b353550675353f46cb99c6f0/cryptography-1.4.tar.gz"
+    sha256 "bb149540ed90c4b2171bf694fe6991d6331bc149ae623c8ff419324f4222d128"
+  end
```

Comment:
Does this also mean the formula now depends on `openssl` and thus needs a `depends_on "openssl"` line? At least that's my interpretation of the build failure where it fails to find an OpenSSL header …